pontoon/sync/vcs/repositories.py

```diff
@@ -327,6 +330,8 @@ def normalize_revision(rev):
             ['svn', 'diff', '-r', '{}:{}'.format(from_revision, 'HEAD'), '--summarize'],
             cwd=path
         )
+        if type(output) is bytes:
```

Comment:
Do you want to drop this on Python 3?

Reply:
I'm not sure, I've seen cases where this was an instance of `str` and I wasn't able to figure out when or why.